config/initializers/foreman.rb

```diff
@@ -36,4 +36,5 @@
 # The users table may not be exist during initial migration of the database
 TopbarSweeper.expire_cache_all_users if (User.table_exists? rescue false)
 
-Foreman::Plugin.medium_providers.register MediumProviders::Default
+Foreman::Plugin.clear
```

Comment:
This implies at this point no plugins could have been loaded. Is that always true?

Reply:
it should be, the plugin initializers are using the `:before => :finisher_hook`, if they wouldn't we would be in bit of a pickle, but they should have for other reasons - the core should be initialized before plugin registers :shrug: